CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+### 0.5.0
+
+FEATURES:
+* [70](https://github.com/nginxinc/nginx-prometheus-exporter/pull/70) Set user agent on scrape requests to nginx.
```

Comment:
```suggestion
* [70](https://github.com/nginxinc/nginx-prometheus-exporter/pull/70): Set user agent on scrape requests to nginx.
```
Please make a similar fix in the rest of the lines. This is for consistency with the existing change-log entries.